ketcher-autotests/tests/Templates/Salts-and-Solvents/check-new-salt-or-solvent-is-replacing.spec.ts

```diff
@@ -24,6 +26,31 @@ test.describe('Salts and Solvents replacement', () => {
     /*
 Test case: EPMLSOPKET-12972 - 'Check that new Salt or Solvent is replacing the previously added one'
   */
+    await selectSaltsAndSolvents(SaltsAndSolvents.AceticAnhydride, page);
+    await clickInTheMiddleOfTheScreen(page);
+
```

Comment:
you have a lot of repeated actions like : 
   await selectSaltsAndSolvents(SaltsAndSolvents.AceticAcid, page);
    await clickInTheMiddleOfTheScreen(page);
    
it might be better to use an array and function to use this array, here's an example:
const atoms = ['H+', 'D', 'T', 'R', 'Pol'];
  for (const atom of atoms) {
    test(`${atom} recognition`, async ({ page }) => {
      // Test case: EPMLSOPKET-1470, 1472, 1480, 1482, 1484
      await selectAtomInToolbar(AtomButton.Extended, page);
      await page.getByRole('button', { name: atom, exact: true }).click();
      await page.getByTestId('OK').click();
      await clickInTheMiddleOfTheScreen(page);

Reply:
Done.